src/webviews/extension-side/serviceRegistry.node.ts

```diff
@@ -27,4 +32,6 @@ export function registerTypes(serviceManager: IServiceManager, _isDevMode: boole
         IJupyterVariableDataProviderFactory,
         JupyterVariableDataProviderFactory
     );
+    serviceManager.add<INotebookExporter>(INotebookExporter, JupyterExporter);
```

Comment:
Would it make more sense to put this into the interactive src tree? It has nothing to do with webviews but at least it depends upon the IW code.